### PR TITLE
[5.3] add description in laravel framework

### PR DESCRIPTION
**Pull request by jsusilo**
i'm add description because i'm often make API Ressfull after my develop project in my website

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![Latest Unstable Version](https://poser.pugx.org/laravel/framework/v/unstable.svg)](https://packagist.org/packages/laravel/framework)
 [![License](https://poser.pugx.org/laravel/framework/license.svg)](https://packagist.org/packages/laravel/framework)
 
-Laravel is a web application framework with expressive, elegant syntax. We believe development must be an enjoyable, creative experience to be truly fulfilling. Laravel attempts to take the pain out of development by easing common tasks used in the majority of web projects, such as authentication, routing, sessions, queueing, and caching.
+Laravel is a web application framework with expressive, elegant syntax. We believe development must be an enjoyable, creative experience to be truly fulfilling. Laravel attempts to take the pain out of development by easing common tasks used in the majority of web projects, such as authentication, restfull api, routing, sessions, queueing, and caching.
```
Comment (zaknes) [inline readme.md]: a restful api*

Comment (jsusilo) [inline readme.md]: owh thanks @zaknes, i'm sorry if my english is bad. i'm first contributor in open source project.
i hope my PR is merged :)

**Comment by taylorotwell** [comment]
It's fine as is.

**Comment by jsusilo** [comment]
oke no problem mr @taylorotwell

**Pull request closed without merging**